### PR TITLE
Added account metadata

### PR DESCRIPTION
**Pull request by wachterjohannes**
__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| Related PRs      | none
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
@wachterjohannes Do you have to rebase that? Can't believe that this should be another 79 files.

**Comment by wachterjohannes** [comment]
haha yes has to be rebased the branch was created from another one

**Comment by wachterjohannes** [comment]
@danrot fixed

File: src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/metadata/list-builder-doctrine-1.0.xsd [hunk 1]
```diff
@@ -6,7 +6,7 @@
             <xs:element type="xs:string" name="field-name"/>
             <xs:element type="xs:string" name="entity-name"/>
 
-            <xs:element type="joinType" name="join" maxOccurs="unbounded" minOccurs="0"/>
+            <xs:element type="joinsType" name="joins" maxOccurs="unbounded" minOccurs="0"/>
```
Comment (wachterjohannes) [inline src/Sulu/Component/Rest/ListBuilder/Metadata/Resources/schema/metadata/list-builder-doctrine-1.0.xsd]: was not used in the contact.

**Comment by wachterjohannes** [comment]
@danrot this is rebased and ready for review

**Pull request merged**